Topics/math-fundamentals.md

```diff
@@ -0,0 +1,172 @@
+
+**In fact:** 
+
+If you've done a second-degree school (you know matrices and derivatives, for example), you probably just need to refresh your memory on the things you already know and go deeper into some things you may not have seen at school. 
+
+In general, if you have a scientific degree (engineering, computer science, physics, etc.), it will probably be just a review, because in this guide _you will learn how learn_ the fundamental theory behind Calculus, Linear Algebra, and Matrix Algebra courses are suggested within this guide.
+
+Whatever your starting level, however, it is still recommended to go through all the resources organized here, _to make sure you do not have "gaps in your knowledge"._
+
+Remember to read and try to implement some ideas of the [**Virgilio's Learning Strategy - Learning to Learn**](teaching.md)
+
+## Courses
+### Calculus
+
+The first macro-topic you need to be confident with is [Calculus](https://en.wikipedia.org/wiki/Calculus).
+
+It would be impossible to list how many Machine Learning system properties are based on its rules, like many of the scientific fields!
+
+The process of training an ML model it's nothing more than Calculus rules applied to statistical models! 
+
+**Lucky you!** 
+You can find great free courses offered by the professors of the [MIT](http://www.mit.edu/)(Massachusetts Institute of Technology), through the index page [MIT OpenCourseWare](https://ocw.mit.edu/index.htm).
+
+The course we suggest in particular is the following:
+
+[**Single Variable Calculus**](https://ocw.mit.edu/courses/mathematics/18-01sc-single-variable-calculus-fall-2010/)
+
+There are both the video lessons and the notes of the course.
+
+Once you've taken the full course, watch this awesome playlist from [3Blue1Brown](https://www.youtube.com/channel/UCYO_jab_esuFRV4b17AJtAw):
+
+[**The Essence of Calculus**](https://www.youtube.com/watch?v=WUvTyaaNkzM&list=PLZHQObOWTQDMsr9K-rj53DwVRMYO3t5Yr)
+
+Thanks to these two resources, you should be able to understand most of the mathematical reasons that underpin today's Machine Learning systems and build them to develop powerful Data Science projects
+
+If you want to dive deeper, here you can find a [Multivariable Calculus](https://en.wikipedia.org/wiki/Multivariable_calculus) course with notes and videos, again from MIT:
+
+[**Multivariable Calculus**](https://ocw.mit.edu/courses/mathematics/18-02sc-multivariable-calculus-fall-2010/)
+
+Once you will feel ready to go deeper inside the mathematics for Machine Learning (definitely not take this at the first pass), here you find your Bible:
+
+[**Mathematics for Machine Learning**](https://mml-book.github.io/)
+
+#### Check your knowledge
+
+Test your knowledge with [**these**](http://webmath2.unito.it/paginepersonali/cordero/English/derivatives.pdf) exercises.
+
+Anyway you can find a ton of them online, and remember, that the more you practise, the better you will get!
+
```

Comment:
For the more academically inclined, they can also refer to [these online notes](http://tutorial.math.lamar.edu/) with very nice explanations, examples and exercises on Calculus and Linear Algebra.